streams/src/main/java/org/apache/kafka/streams/kstream/HoppingWindows.java

```diff
@@ -23,14 +23,16 @@
 import java.util.Map;
 
 /**
- * The hopping window specifications used for aggregations.
+ * The hopping window specifications used for aggregations. TODO extend
  */
 public class HoppingWindows extends Windows<HoppingWindow> {
 
     private static final long DEFAULT_SIZE_MS = 1000L;
 
+    /** The size of the window in milliseconds. */
     public final long size;
```

Comment:
The size / period are defined on event time, and hence not necessarily Ms, we should instead remove default values and any `ms` references in those specs. The removal is already done in your PR.